<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page2nd</class>
 <widget class="QWidget" name="Page2nd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: white;
 }</string>
  </property>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>25</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid #FFA25C;
    background-color: white;
    padding: 4px;
    border-radius: 4px;
    outline: none; 
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemShutdown"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cctv">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>451</width>
     <height>254</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #FFA25C;
border-radius: 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="floor_table_view">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>621</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
	border: 2px solid #FFA25C;
	border-radius: 4px;
    selection-background-color: #FFDDAC;   /* 연한 주황색 선택 배경 */
    selection-color: black;               /* 선택 시 글자색 */
    alternate-background-color: #FFF3E0;  /* 홀짝 줄 배경: 연한 살구색 */
    background-color: white;
    gridline-color: #FFBB70;              /* 그리드 라인도 주황 계열 */
}

QHeaderView::section {
    background-color: #FFD08A;            /* 헤더 배경: 주황 계열 */
    padding: 4px;
    font-weight: bold;
    border: 1px solid #FFBB70;            /* 헤더 테두리 주황 */
}
</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="mini_map_view">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>621</width>
     <height>371</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #FFA25C;
border-radius: 4px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>25</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid #FFA25C;
    background-color: white;
    padding: 4px;
    border-radius: 4px;
    outline: none; 
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_floor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>45</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
    border: 2px solid #FFA25C;
    background-color: white;
    padding: 4px;
    border-radius: 4px;
    font-size: 22px;
}
</string>
   </property>
   <property name="text">
    <string>B1F</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cctv_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>45</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
    border: 2px solid #FFA25C;
    background-color: white;
    padding: 4px;
    border-radius: 4px;
    font-size: 22px;
}
</string>
   </property>
   <property name="text">
    <string>CCTV</string>
   </property>
  </widget>
  <widget class="QListWidget" name="logList">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #FFA25C;</string>
   </property>
  </widget>
  <zorder>exit_button</zorder>
  <zorder>label_cctv</zorder>
  <zorder>floor_table_view</zorder>
  <zorder>logout_button</zorder>
  <zorder>mini_map_view</zorder>
  <zorder>label_floor</zorder>
  <zorder>label_cctv_2</zorder>
  <zorder>logList</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
